target/msm8226/init.c

```diff
@@ -228,33 +230,53 @@ void target_sdc_init()
 {
 	struct mmc_config_data config = {0};
 
-	/*
-	 * Set drive strength & pull ctrl for emmc
-	 */
-	set_sdc_power_ctrl();
-
+#ifndef FORCE_SDCARD
+	/* Try slot 1*/
 	config.bus_width = DATA_BUS_WIDTH_8BIT;
-	config.max_clk_rate = MMC_CLK_200MHZ;
-
-	/* Trying Slot 1*/
```

Comment:
Ooof, sad diff (try slot 1 vs trying slot 1) but at this point I'm not worrying too much anymore about the old branch :)